docs/get-started/overview.rst

```diff
@@ -99,15 +90,17 @@ what people would actually want to see (e.g. meters-per-second instead of
 kilometers-per-hour or miles-per-hour).
 
 To facilitate working with these quantities, stravalib makes use of the
-`pint library <https://pypi.org/project/Pint/>`_.  You can simply cast the values string
-to see a representation that includes the units::
+`pint library <https://pypi.org/project/Pint/>`_.  You can use the ``quantity()``
+accessor to turn the "plain" ``int`` or ``float`` values into a
+:class:`pint.Quantity` object.::
 
    activity = client.get_activity(96089609)
-   print(activity.distance)
-   # 22530.80 m
+   print(activity.distance.quantity())
+   # 22530.80 meter
 
-Hmmm, meters.  Well, here in the US we like to see miles.  While you can certainly do this using the units
-library directly, stravalib provides a preconfigured set of common units to simplify matters.::
+Hmmm, meters.  Well, here in the US we like to see miles.  While you can certainly do
```

Comment:
@jsamoocha i'm just noticing this language! should we consider making it less USA centric? I don't mind  being a US native. BUT you and others may prefer meters.  🙃 so we could either leave as is or make it a bit less country agnostic. 

like 

> Here, the value is returned in meters. If you prefer units in miles, you can use stravalib's preconfigured set of community units...

As written it's fun and light -- so you tell me if you want to drop the "here in the US to make it more agnostic" or not! :)

Reply:
I don't see any problem with that language, so let's not get too formal on that 😁.